egs/wsj/s5/steps/nnet3/decode_lookahead.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+# Copyright 2012-2015  Johns Hopkins University (Author: Daniel Povey).
+# Apache 2.0.
+
+# This script does decoding with a neural-net with lookahead composition of HCL and G graphs.
```

Comment:
please fix author in copyright info.
If possible an example script for a smaller setup might be easier to demo to people (eg mini_librispeech) but not necessary.

Reply:
Sure, I will add mini version too.

Reply:
Done, mini_librispeech also added